chsdi/models/vector/uvek.py

```diff
@@ -393,6 +393,88 @@ class unf_pers_velo(Base, Vector):
 register('ch.astra.unfaelle-personenschaeden_fahrraeder', unf_pers_velo)
 
 
+class schwerverunf_kanton_alkohol(Base, Vector):
+    __tablename__ = 'unf_schwer'
+    __table_args__ = ({'schema': 'astra', 'autoload': False, 'extend_existing': True})
+    __template__ = 'templates/htmlpopup/astra_schwerverunf_kanton_alkohol.mako'
+    __bodId__ = 'ch.astra.schwerverunfallte-kanton_alkohol'
+    __label__ = 'canton'
+    id = Column('bgdi_id', Integer, primary_key=True)
+    the_geom = Column(Geometry(geometry_type='GEOMETRY',
+                               dimension=2, srid=21781))
```

Comment:
Queryable attributes here? Related to https://github.com/geoadmin/mf-geoadmin3/pull/3244/files